contentgrid-spring-audit-logging/src/main/java/com/contentgrid/spring/audit/handler/LoggingAuditHandler.java

```diff
@@ -0,0 +1,93 @@
+package com.contentgrid.spring.audit.handler;
+
+import com.contentgrid.spring.audit.event.AbstractEntityAuditEvent;
+import com.contentgrid.spring.audit.event.AbstractEntityItemAuditEvent;
+import com.contentgrid.spring.audit.event.AbstractAuditEvent;
+import com.contentgrid.spring.audit.event.BasicAuditEvent;
+import com.contentgrid.spring.audit.event.EntityContentAuditEvent;
+import com.contentgrid.spring.audit.event.EntityItemAuditEvent;
+import com.contentgrid.spring.audit.event.EntityRelationAuditEvent;
+import com.contentgrid.spring.audit.event.EntitySearchAuditEvent;
+import java.util.Locale;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.event.Level;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+
+@Slf4j
+public class LoggingAuditHandler implements AuditEventHandler {
+
+    @Override
+    public void handle(AbstractAuditEvent auditEvent) {
+        var auditLogBuilder = log.atLevel(Level.INFO);
+
+        StringBuilder logMessage = new StringBuilder("{} {} -> {}");
+
+        auditLogBuilder =auditLogBuilder.addArgument(auditEvent.getRequestMethod())
+                .addKeyValue("requestMethod", auditEvent.getRequestMethod())
+                .addArgument(auditEvent.getRequestUri())
+                .addKeyValue("requestUri", auditEvent.getRequestUri())
+                .addArgument(auditEvent.getResponseStatus())
+                .addKeyValue("responseStatus", auditEvent.getResponseStatus())
```

Comment:
I have a personal preference to use `http.method`, `http.uri` and `http.status` instead. WDYT ?

Reply:
For the keys in the logging?